Makefile

```diff
@@ -402,11 +402,17 @@ generate-e2e-templates-v1.10: $(KUSTOMIZE)
 	"$(KUSTOMIZE)" --load-restrictor LoadRestrictionsNone build "$(E2E_GOVMOMI_TEMPLATE_DIR)/v1.10/clusterclass" > "$(E2E_GOVMOMI_TEMPLATE_DIR)/v1.10/clusterclass-quick-start.yaml"
 	"$(KUSTOMIZE)" --load-restrictor LoadRestrictionsNone build "$(E2E_GOVMOMI_TEMPLATE_DIR)/v1.10/workload" > "$(E2E_GOVMOMI_TEMPLATE_DIR)/v1.10/cluster-template-workload.yaml"
 
+	"$(KUSTOMIZE)" --load-restrictor LoadRestrictionsNone build "$(E2E_SUPERVISOR_TEMPLATE_DIR)/v1.10/clusterclass" > "$(E2E_SUPERVISOR_TEMPLATE_DIR)/v1.10/clusterclass-quick-start-supervisor.yaml"
+	"$(KUSTOMIZE)" --load-restrictor LoadRestrictionsNone build "$(E2E_SUPERVISOR_TEMPLATE_DIR)/v1.10/workload" > "$(E2E_SUPERVISOR_TEMPLATE_DIR)/v1.10/cluster-template-workload-supervisor.yaml"
```

Comment:
Note: these currently renders the kustomize / yaml files with a copy of the templates on main.

We propably should use the ones from `release-1.10`?

(copy over from here https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/tree/release-1.10/test/e2e/data/infrastructure-vsphere-supervisor/main )

Reply:
Note: there's no real diff

Reply:
I think it's okay. For supervisor at this point it's not 100% important that we test the ones we released. It's just important that they work with the old version